awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroup/InventoryGroup.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useState } from 'react';
+import { t } from '@lingui/macro';
+import { withI18n } from '@lingui/react';
+import { CardHeader } from '@patternfly/react-core';
+
+import { Switch, Route, withRouter, Link, Redirect } from 'react-router-dom';
+import { GroupsAPI } from '@api';
+import CardCloseButton from '@components/CardCloseButton';
+import RoutedTabs from '@components/RoutedTabs';
+import ContentError from '@components/ContentError';
+import ContentLoading from '@components/ContentLoading';
+import InventoryGroupEdit from '../InventoryGroupEdit/InventoryGroupEdit';
+import InventoryGroupDetail from '../InventoryGroupDetail/InventoryGroupDetail';
+
+function InventoryGroups({ i18n, match, setBreadcrumb, inventory, history }) {
+  const [inventoryGroup, setInventoryGroup] = useState(null);
+  const [hasContentLoading, setContentLoading] = useState(true);
+  const [hasContentError, setHasContentError] = useState(false);
+
+  useEffect(() => {
+    const loadData = async () => {
+      try {
+        const { data } = await GroupsAPI.readDetail(match.params.groupId);
+        setInventoryGroup(data);
+        setBreadcrumb(inventory, data);
+      } catch (err) {
+        setHasContentError(err);
+      } finally {
+        setContentLoading(false);
+      }
+    };
+
+    loadData();
+  }, [match.params.groupId, setBreadcrumb, inventory]);
```

Comment:
It is unclear to me why we need to include `setBreadcrumb` in this list. Shouldn't `match.params.groupId` and `inventory` be enough?

Reply:
The linter throws a warning that says we need to keep it in.  I agree that it seems weird that we need to keep it in there. I feel like we are on the verge of having to re think how we are creating our breadcrumbs because it `/inventories` we have to do a lot of prop drilling for `setBreadcrumb`

Reply:
sounds good to me - let's leave it in.